portal-api/src/v1/helpers/escapeOperators.js

```diff
@@ -70,9 +71,15 @@ const recursivelyReplaceEscapeOperators = (filters, result = {}) => {
     } else if (typeof filters[key] === 'object' && filters[key] !== null) {
       result[key] = {};
       recursivelyReplaceEscapeOperators(filters[key], result[key]);
-      // These last two if statements handle the lowest level cases
+      // These last three if statements handle the lowest level cases
```

Comment:
```suggestion
      // These last few statements handle the lowest level cases
```

Could be very easy to miss this if a statement is added or removed (good spot)